app/Http/Controllers/MemberController.php

```diff
@@ -12,10 +12,29 @@ class MemberController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
         $members = Member::all();
-        return response()->json($members);
+        $messages = [];
+        $action = $request->get('ACTION');
```

Comment:
I think you should do this following the REST standard (using http methods):
Create = POST
Read = GET
Update = PUT/PATCH
Delete = DELETE

You can specify 1 method for create, 1 method for update, etc.. And bind them to the method and uri in your routes.php file